jbpm-console-ng-case-mgmt/jbpm-console-ng-case-mgmt-api/src/main/java/org/jbpm/console/ng/cm/model/CaseCommentSummary.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.console.ng.cm.model;
+
+import java.util.Date;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jbpm.console.ng.ga.model.GenericSummary;
+
+@Portable
+public class CaseCommentSummary extends GenericSummary {
+
+    private String caseId;
+    private String author;
+    private String text;
+    private Date addedAt;
+
+    public CaseCommentSummary() {
+    }
+
+    public CaseCommentSummary(final String caseId, final String id, final String author, final String text, final Date addedAt) {
+        super(id,null);
+        this.caseId=caseId;
+        this.author = author;
+        this.text = text;
+        this.addedAt = addedAt;
+    }
+
+    public String getCaseId() {
+        return caseId;
+    }
+
+    public void setCaseId(String caseId) {
+        this.caseId = caseId;
+    }
+
+    public String getAuthor() {
+        return author;
+    }
+
+    public void setAuthor(String author) {
+        this.author = author;
+    }
+
+    public String getText() {
+        return text;
+    }
+
+    public void setText(String text) {
+        this.text = text;
+    }
+
+    public Date getAddedAt() {
+        return addedAt;
+    }
+
+    public void setAddedAt(Date addedAt) {
+        this.addedAt = addedAt;
+    }
+
+    @Override
+    public String toString() {
+        return "CaseCommentSummary{" +
+                "caseId=" + caseId +
+                ", id=" + id +
```

Comment:
@cristianonicolai Ah, ok, I missed the `super.toString()` when I looked the first time. Anyway I find it weird to implement `toString()` in this way, because the resulting case comment will be stringified like `"CaseCommentSummary{caseId=x, author=y, text=z, addedAt=14/09/2016} GenericSummary{id=123, name='something'}"` which is confusing to say the least. I'd rather see String like "CaseCommentSummary{id=123, name='something', caseId=x, author=y, text=z, addedAt=14/09/2016}".

However if we're not using it anywhere in the UI it's ok to leave it like this.